src/tests/system/tests/test_ipa_trusts.py

```diff
@@ -60,3 +61,70 @@ def test_ipa_trusts__lookup_group_without_sid(ipa: IPA, trusted: GenericADProvid
     status = ipa.sssctl.domain_status(trusted.domain, online=True)
     assert "online status: offline" not in status.stdout.lower(), "AD domain went offline!"
     assert "online status: online" in status.stdout.lower(), "AD domain was not online!"
+
+
+@pytest.mark.importance("critical")
+@pytest.mark.topology(KnownTopologyGroup.IPATrust)
+def test_ipa_trusts__authentication_with_default_settings(client: Client, ipa: IPA, trusted: GenericADProvider):
+    """
+    :title: Authenticate IPA and trusted AD users with default settings
+    :setup:
+        1. Create users
+        2. Start SSSD
+    :steps:
+        1. Authenticate users, one extra time using the fully qualified name for the ipa user
+        2. Authenticate users using the wrong password
+    :expectedresults:
+        1. Logins are successful
+        2. Logins are unsuccessful
+    :customerscenario: False
+    """
+    ipa_user = ipa.user("user1").add(password="Secret123").name
```

Comment:
Here you are creating a regular IPA user, not a trusted IPA user, not sure if this was intentional or not.

Reply:
It is intentional, a "control" user.  Old habits, I'll remove it.